Extensions/Leaderboards/JsExtension.js

```diff
@@ -28,13 +28,13 @@ module.exports = {
     extension
       .setExtensionInformation(
         'Leaderboards',
-        _('Leaderboards'),
+        _('Leaderboards (experimental)'),
         _('Allow your game to send scores to your leaderboards.'),
         'Florian Rival',
         'Open source (MIT License)'
       )
-      .setCategory('Leaderboards');
-    extension
+      .setExtensionHelpPath('/all-features/leaderboards')
+      .setCategory('Leaderboards')
```

Comment:
A category is meant to be generic and applicable to multiple extensions. Maybe something like "GDevelop services", "Online services" or "Liluo services" would be better?